js/src/api/util/address.js

```diff
@@ -0,0 +1 @@
+../../abi/util/address.js
```

Comment:
`~/abi/util` is not part of the exposed interfaces (i.e. stuff on Abi.*), so don't use it. 

Could kick the can for this further down the road and not fix it here since this part will break when that is eventually changed, but since this this whole PR is a non-critical addition (especially when it comes to sources for 3rd party), don't see the need to try rush without doing things properly.